README.md

```diff
@@ -0,0 +1,106 @@
+# Linux GPIO Utils
+
+GPIO Utils provides convenient access to GPIOs on a Linux system. The library
+builds on top of the sysfs interface to GPIOs exposed by the kernel and provides
+essential functionality required for most embedded systems.
+
+## Install
+
+To install the latest released version of gpio utils, ensure that you have
+installed Rust and then run:
+
+```sh
+cargo install gpio-utils
+```
+
+## Features
+
+- [ ] Infrastructure for providing names for the GPIOs in one systems providing
+      names that map to individual pins.  These names (in addition to GPIO numbers)
+      may be used with other commands.
+- [ ] Ability to export/unexport GPIOs and expose symlinks using the GPIO "friendly"
+      names simpler.
+- [ ] Ability to set input/output state on each pin
+- [ ] Ability to get/set gpio values by pin number or name (including temporary
+      export if necessary)
+- [ ] Ability to block awaiting pin state change (with timeout)
+
+## System Integration
+
+GPIO Utils provides two main pieces that one may integrate into their final
+system:
+
+1. The `gpio` command.  This provides the core functionality for GPIO Utils and
+   is useful in its own right.
+2. The `gpio` init script.  This init script can be integrated into a target
```

Comment:
Yes, should add that.  I would like this to be easily integrated into distros.  Would be cool to, e.g., see this in debian and friends.